NREL/ssc
# Cmod utilityrateforecast

Creates a new cmod so that code utilized by price signals dispatch can be called from PySAM. Plan is for TEMPO team to use time-series prices from this cmod to calculate optimized charging profiles for inputs to dGen.

Companion PRs:
SAM: https://github.com/NREL/SAM/pull/1144
PySAM: https://github.com/NREL/pysam/pull/119

To test:
- Check out these branches and build PySAM
- Run new UtilityRateForecastExample.py and examine outputs
